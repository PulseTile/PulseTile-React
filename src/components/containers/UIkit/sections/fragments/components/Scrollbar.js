import React from 'react';

/**
 * This component returns content of Scrollbar section
 */
const Scrollbar = () => {
  return (
    <div id="scrollbar" className="ui-section">
      <strong className="ui-title">Scrollbar</strong>
      <div className="ui-scrollbar">
        <div className="ui-scrollbar-content"></div>
      </div>
    </div>
  );
};

export default Scrollbar;
