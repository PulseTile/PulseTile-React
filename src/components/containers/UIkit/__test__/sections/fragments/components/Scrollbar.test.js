import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Scrollbar from '../../../../sections/fragments/components/Scrollbar';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Scrollbar />', () => {
  it('should renders Scrollbar with props correctly', () => {
    const component = shallow(<Scrollbar />);
    expect(component).toMatchSnapshot();
  });
});
