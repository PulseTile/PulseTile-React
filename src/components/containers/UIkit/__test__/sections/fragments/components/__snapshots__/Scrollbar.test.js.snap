// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Scrollbar /> should renders Scrollbar with props correctly 1`] = `
<div
  className="ui-section"
  id="scrollbar"
>
  <strong
    className="ui-title"
  >
    Scrollbar
  </strong>
  <div
    className="ui-scrollbar"
  >
    <div
      className="ui-scrollbar-content"
    />
  </div>
</div>
`;
